### DISCUSSION TITLE
Is it possible to use the climate add-on to steer/control another relay within the HA environment?

### DISCUSSION THREAD

**Discussion by 4irm** [Blackymas/NSPanel_HA_Blueprint#2336] [Q&A] [answered]
I mean not using the internal relay 1 or 2, instead using a whatever entity in HA.

Thanks!

**Comment by edwardtfn** [comment]
For this I would suggest NOT installing the climate add-on, but instead using the [Generic Thermostat](https://www.home-assistant.io/integrations/generic_thermostat/) available in Home Assistant, which cn use your panel's temperature sensor or not, but can be assigned as the climate entity of your panel, so you can use the panel to control the generic thermostat.

**Reply by 4irm** [reply]
> For this I would suggest NOT installing the climate add-on, but instead using the [Generic Thermostat](https://www.home-assistant.io/integrations/generic_thermostat/) available in Home Assistant, which can use your panel's temperature sensor or not, but can be assigned as the climate entity of your panel, so you can use the panel to control the generic thermostat.

Thanks for your feedback. I am not sure if I fully understand. I do have this scenario...

Room 1) NSPANEL 1 > Should switch/control the relay for the underfloor heater of room 2. The wires for the heater of room 2 are available on the NSPANEL 1. No interface for temperature regulation on the NSPANEL 1 itself is needed.

Room 2) NSPANEL 2 > Does not switch/control the relay, however, uses the climate add-on (on the touchscreen) to set the temperature for NSPANEL 1 relay.

I know a strange setup, but this is because the wiring is setup in this way...How would that work 

Thanks.

**Reply by edwardtfn** [reply, marked as answer]
That's not strange at all. 😉 

On NSPanel 1, where the relays are used to physically control your heater, your should include the add-on climate, as that will allow the local control of the heating even when the Wi-Fi is out.
That will create a climate entity on the device's page under Home Assistant (Settings > Devices & services > ESPHome). When setting up the blueprint for NSPanel 1, please select that climate entity as your main climate entity for that panel.

On NSPanel 2, please don't use that add-on, as that is not needed. You can set the panel normally, without that add-on, but when configuring the blueprint, please also select the climate entity related to NSPanel 1, so you will be able to, from NSPanel 2, control the settings of the thermostat of NSPanel 1. This will however need the Wi-Fi and Home Assistant to be available to both panels.

Please let me know how that works.
